opencti-platform/opencti-graphql/src/modules/internal/csvMapper/csvMapper-utils.ts

```diff
@@ -39,19 +45,30 @@ const representationLabel = (idx: number, representation: CsvMapperRepresentatio
   return `${number} ${representation.target.entity_type}`;
 };
 
-export const parseCsvMapper = (entity: any): CsvMapperParsed => {
+export const parseCsvMapper = (mapper: any): CsvMapperParsed => {
+  let representations: CsvMapperRepresentation[] = mapper?.representations ?? [];
+  if (typeof mapper?.representations === 'string') {
```

Comment:
That's confuse me to see a variable where we explicitly tell the type is `CsvMapperRepresentation[]` and the line after we check if the content of the variable is a string.

Should we not instead initialize the variable `representations` to nothing and fill it only where we are sure of the type of `mapper.representations`?